AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -139,31 +223,150 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
         languageSelected = "en-US"
         regionSelected = "auto"
+        partialResultsStabilizationSelected = 0
+        piiContentIdentificationSelected = ""
+        piiContentRedactionSelected = ""
         
         engineTextField.text = enginesDict[engineSelected]
         languageTextField.text = languagesDict[languageSelected]
         regionTextField.text = regionsDict[regionSelected]
-        
+        partialResultsStabilizationTextField.text = partialResultsStabilizationsList[partialResultsStabilizationSelected]
+        piiContentIdentificationTextField.text = piiContentIdentificationDict[piiContentIdentificationSelected]
+        piiContentRedactionTextField.text = piiContentRedactionDict[piiContentRedactionSelected]
+    
         enginePickerView.delegate = self
         languagePickerView.delegate = self
         regionPickerView.delegate = self
         enginePickerView.dataSource = self
         languagePickerView.dataSource = self
         regionPickerView.dataSource = self
+        partialResultsStabilizationPickerView.delegate = self
+        partialResultsStabilizationPickerView.dataSource = self
+        piiContentIdentificationPickerView.delegate = self
+        piiContentIdentificationPickerView.dataSource = self
+        piiContentRedactionPickerView.delegate = self
+        piiContentRedactionPickerView.dataSource = self
         
         enginePickerView.tag = 1
         languagePickerView.tag = 2
         regionPickerView.tag = 3
+        partialResultsStabilizationPickerView.tag = 4
+        piiContentIdentificationPickerView.tag = 5
+        piiContentRedactionPickerView.tag = 6
+        
+        storyBoard = UIStoryboard(name: "Main", bundle: nil)
+        languageDetectionViewController = storyBoard!.instantiateViewController(withIdentifier: "languageDetection") as? AutomaticLanguageDetectionViewController
+        
+        languageDetectionViewController?.cancellationHandler = { numberOfselectedLanguages in
+            self.enableAutomaticLanguageDetectionSwitch.isOn = numberOfselectedLanguages! > 1
+        }
+        
+        languageDetectionViewController?.selectedLanguageOptionsHandler = { selectedLanguages in
+            self.selectedLanguagesDetectionOptions = selectedLanguages!
+        }
+        
+        languageDetectionViewController?.preferredLanguageOptionHandler = { preferredLanguage in
+            self.preferredLanguagesDetectionOptions = preferredLanguage!
+        }
+        
+        self.languageHandler = { languageList in
+            self.languageDetectionViewController?.languages = languageList!
+        }
+        
+        self.languagesDictHandler = { mappings in
+            self.languageDetectionViewController?.languagesDict = mappings!
+        }
     }
     
     @IBAction func startTranscriptionButton(_ sender: UIButton) {
-        let url = self.meetingEndpointUrl.hasSuffix("/") ? self.meetingEndpointUrl : "\(self.meetingEndpointUrl)/"
```

Comment:
This is latest, should not change in rebase. The url will be passed from the func call, not from AppConfiguration.url any more.

Reply:
revered to latest in my next commit.